AzureDevOpsTemplates/Build/StepTemplates/dfc-arm-build.yml

```diff
@@ -0,0 +1,44 @@
+parameters:
+  ArmTemplateRoot: ''
+  SolutionBaseName: ''
+  AzureSubscription: 'SFA-CDH-Dev/Test (962cae10-2950-412a-93e3-d8ae92b17896)'
+  PublishPipelineArtifact: false
+  TestArmTemplateScriptUrl: https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/PSScripts/Test-ArmTemplate.ps1
+  TestArmTemplateScriptFilePath: $(System.DefaultWorkingDirectory)\Test-ArmTemplate.ps1
+  TokenizeTestParameters: false
+
+steps:
+- task: esfadevops.Tokenization.custom-build-task.Tokenization@0
```

Comment:
Do we ever have tokens in our test-parameters.yml files?  I can't recall seeing any...

Reply:
I meant test-parameters.json...

Reply:
Update: Yes we do!